pycbc/waveform/nr_waveform.py

```diff
@@ -181,10 +179,11 @@ def get_param(value):
             curr_h_real = curr_amp * numpy.cos(curr_phase)
             curr_h_imag = curr_amp * numpy.sin(curr_phase)
             curr_ylm = lal.SpinWeightedSphericalHarmonic(theta, phi, -2, l, m)
-            hp += curr_h_real * curr_ylm.real - curr_h_imag * curr_ylm.imag
+            print curr_ylm
+            hp += curr_h_real * curr_ylm.real + curr_h_imag * curr_ylm.imag
             # FIXME: No idea whether these should be minus or plus, guessing
             #        minus for now. Only affects some polarization phase
-            hc += - curr_h_real * curr_ylm.imag - curr_h_imag * curr_ylm.real 
+            hc += + curr_h_real * curr_ylm.imag - curr_h_imag * curr_ylm.real 
```

Comment:
Ian: I redid our calculation from the summer and I got the same result. However, I think for consistency, the convention should be h_{lm} = |h_{lm}| * exp(-i phase_{lm})
and that would change the signs in the second terms in hp and hc. So, hp would be consistent line 183 but hc in line 186 would not be the same as yours.